src/styles/themes.css

```
@@ -1,65 +1,67 @@
+	--frame-edge: rgba(26, 26, 26, 0.05);
+	--accent: #b85c38;
+	--accent-2: #2a6b5a;
+	--accent-3: #6c6259;
+	--accent-4: #1a1a1a;
+	--line-strong: rgba(26, 26, 26, 0.1);
+	--shadow-color: rgba(26, 26, 26, 0.06);
+	--panel-shadow: 0 18px 42px rgba(26, 26, 26, 0.08);
+	--panel-shadow-soft: 0 1px 3px rgba(26, 26, 26, 0.05), 0 4px 12px rgba(26, 26, 26, 0.06);
+	--selection-bg: #b85c38;
+	--selection-fg: #ffffff;
+	--font-body: 'Inter', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)